modules/ROOT/pages/alerts.adoc

```diff
@@ -81,33 +70,27 @@ You can enable or disable an alert by toggling the <<alerts, switch>> on the rig
 Depending on the value, the alert transitions to `OK` or `Alerting`.
 * Disabling an alert causes the alert to transition to `Disabled`.
 
-== Configure API Manager and Runtime Manager Alerts
-
-From the *Alerts* page in Anypoint Monitoring, you can access configuration pages for API Manager and Runtime Manager alerts.
-
-Depending on your subscription level:
-
-* After selecting *New alert*, you can opt to create an alert in API Manager or Runtime Manager:
-
-** *Create an alert for your API in API Manager*
-+
-For guidance about alerts in API Manager, see xref:2.x@api-manager::using-api-alerts.adoc[About API Alerts]. Note that these alerts are not listed in the *Alerts* page for Anypoint Monitoring.
-** *Create an alert for your server in Runtime Manager*
-+
-For guidance about alerts in Runtime Manager, see xref:runtime-manager::alerts-on-runtime-manager.adoc[About Configuring Alerts Using Runtime Manager]. Note that these alerts are not listed in the *Alerts* page for Anypoint Monitoring.
-* If you don't have a Titanium subscription, you might see *https://www.mulesoft.com/anypoint-pricing[Titanium subscription required]*.
+[[other-alerts]]
+== Add API Manager and Runtime Manager Alerts
 
-Instead of seeing links to API and runtime alerts, you receive visual guidance showing how to open the configuration area for alerts in graphs for custom dashboards.
+In addition to creating basic and advanced alerts in Anypoint Monitoring, you can also navigate from Anypoint Monitoring to alert creation pages in API Manager and Runtime Manager. 
 
-== Add API Manager and Runtime Manager Alerts
+image::alerts-menu.png["Alerts menu options to set up basic, advanced, or other types of alerts"]
 
-Click *+ New alert* to access API Manager and Runtime Manager alert configuration pages. You can also get basic instructions for configuring alerts for a chart (widget) in a custom dashboard.
+//TODO: VERIFY NOTE
+NOTE: Alerts created in API Manager and Runtime Manager aren't listed in Anypoint Monitoring *Alerts*.
 
-image::alerts-menu.png["Alerts menu options to set up, basic, advanced, or other types of alerts", 40%, 40%]
+. From *Alerts* in Anypoint Monitoring, click *+ New alert*.
```

Comment:
I think we don't include punctuation from UI elements.

Reply:
Hmm, I think the + means Add. I think it's necessary so that it matches the button text.

Reply:
addressed some alt text. for the rest: [W-17361464](https://gus.lightning.force.com/a07EE000026JsovYAC)